### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Shree-Gillorkar**
Added Relational Model in the Table of Contents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Please submit a pull request if there is any material that you think should be i
     * [Hash Table](#hash-table)
     * [Bloom Filter](#bloom-filter)
 * [DDL](#ddl)
+* [Relational Model](#relational-model)
+    * [Codd's Rules](codd's-rules)
+    * [Relational Data Model](relational-data-model)
+    * [Relational Algebra](relational-algebra)
+    * [ER to Relational Model](er-to-realtional-model)
+    * [SQL - Overview](sql---overview)
```
Comment (zz-jason) [inline README.md:41]: @Shree-Gillorkar thanks for your contribution, could you add some paper or books about these topics?

**Comment by Shree-Gillorkar** [comment]
@zz-jason , Yes sure i will do it.

**Comment by Shree-Gillorkar** [comment]
@zz-jason, I added some blogs related to the topic can you please have a look, & please let me know if any changes are required.

**Review by ngaut — APPROVED** [review]
LGTM, thank you.

**Pull request merged**